### PR TITLE
[Mgmt] Inital checkin of MachineLearning

### PR DESCRIPTION
**Pull request by isra-fel**
# Description
This is the initial generation of Azure MachineLearning RP which we want to support in the upcoming preview.

# Checklist

To ensure a quick review and merge, please ensure:
- [x] The PR has a understandable title and description explaining the _why_ and _what_.
- [ ] The PR is opened in draft if not ready for review yet.
   - If opened in draft, please allocate sufficient time (24 hours) after moving out of draft for review
- [x] The branch is recent enough to not have merge conflicts upon creation.

# Ready to Land?
- [ ] Build is completely green
   - Submissions with test failures require tracking issue and approval of a CODEOWNER
- [ ] At least one +1 review by a CODEOWNER
- [ ] All -1 reviews are confirmed resolved by the reviewer 
   - Override/Marking reviews stale must be discussed with CODEOWNERS first

### CODE REVIEW

File: samples/Azure.ResourceManager.MachineLearning/readme.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  Usages: Locations
+  VirtualMachineSizes: Locations
+  Quotas: Locations
+  Notebooks: Workspaces
+  StorageAccount: Workspaces
+directive:
+  - from: swagger-document
+    where: $.definitions.ComputeNodesInformation.properties
+    transform: delete $.nextLink;
+    reason: Duplicated "nextLink" property defined in schema 'AmlComputeNodesInformation' and 'ComputeNodesInformation'
```
Comment (isra-fel) [inline samples/Azure.ResourceManager.MachineLearning/readme.md:49]: Though currently worked around by directive, we are going to let service team know this issue

**Comment by chamons** [comment]
With 460 files and 59k added lines, this is a bit hard to go through on GH.

Here's the diff with no generated files: https://gist.github.com/chamons/3f9c1a84ddba702adfd61286ec781812

And here's that same diff without the very long swagger file included: https://gist.github.com/chamons/43503a23ce34de48b871af23a237b1ea (142 lines of code to review).

**Review by chamons — APPROVED** [review]
Approved from the shared code angle.

**Comment by isra-fel** [comment]
Thanks! I'll try to split them into commits next time ;)

**Review by allenjzhang — APPROVED** [review]

**Pull request closed without merging**